### DISCUSSION TITLE
Missing argocd-redis-role.yaml?

### DISCUSSION THREAD

**Discussion by disposab1e** [argoproj/argo-cd#7050] [Q&A]
I wonder because in 2.1.0 the argocd-redis-role.yaml is missing? Is this intentional? Without this role upgrading in OpenShift fails.

**Comment by chetan-rns** [comment]
Hi @disposab1e, it was removed as part of https://github.com/argoproj/argo-cd/pull/5863. Could you please share more information about the failure? I was able to update Argo CD from 2.05 to 2.1 on OpenShift 4.7

**Reply by disposab1e** [reply]
Do you have deleted the existing 2.0.5 role after ugrade?

**Reply by disposab1e** [reply]
This [RoleBinding](https://github.com/argoproj/argo-cd/blob/v2.1.0/manifests/base/redis/argocd-redis-rolebinding.yaml) makes no sense without a Role. And a [Deployment](https://github.com/argoproj/argo-cd/blob/v2.1.0/manifests/base/redis/argocd-redis-deployment.yaml) with `runAsUser: 999` is nothing OpenShift loves.

**Reply by chetan-rns** [reply]
I didn't delete the existing 2.0.5 role. But I agree the rolebinding is obsolete if the role is missing

**Reply by wtam2018** [reply]
It looks like this is another regression introduced by #5863.  @disposab1e  Do you mind raising an issue?    Thanks.
